Add interfaces/font-metrics-api.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/font-metrics-api.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166

foolip commented:
This should remain open until there's some engine about to implement this. At that time, a test to revive may be found in https://github.com/web-platform-tests/wpt/pull/12950.

stephenmcgruer commented:
As of 2020/04/10, I believe there is still no implementer of this API.

autofoolip committed:
Add interfaces/font-metrics-api.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/font-metrics-api.idl
Build: https://travis-ci.org/w3c/webref/builds/182613166
